src/defaults.sh

```diff
@@ -19,8 +19,8 @@
 [ -z "$IECN" ] && IECN="ECN"
 [ -z "$EECN" ] && EECN="ECN"
 # These two used to be called something else; preserve backwards compatibility
-[ -z "$ZERO_DSCP" ] && ZERO_DSCP="${SQUASH_DSCP:-1}"
-[ -z "$IGNORE_DSCP" ] && IGNORE_DSCP="${SQUASH_INGRESS:-1}"
+[ -z "$ZERO_DSCP_INGRESS" ] && ZERO_DSCP_INGRESS="${SQUASH_DSCP:-1}"
+[ -z "$IGNORE_DSCP_INGRESS" ] && IGNORE_DSCP_INGRESS="${SQUASH_INGRESS:-1}"
```

Comment:
Ah, but now there's no backwards compatibility with the previous change. The easiest is probably to keep the old variable for now and just do a conditional assignment to the _INGRESS version...